ckanext/harvest/logic/action/update.py

```diff
@@ -441,9 +441,15 @@ def harvest_objects_import(context, data_dict):
     return last_objects_count
 
 
-def _calculate_next_run(frequency):
+def _calculate_next_run(frequency, time):
 
     now = datetime.datetime.utcnow()
+    if time and frequency != 'ALWAYS':
+        t = datetime.datetime.strptime(time, '%I:%M %p')
+        set_hour = int(t.strftime("%H"))
+        set_minute = int(t.strftime("%M"))
```

Comment:
Looks like we're only setting hours, do we need the minute? It's always 0 like others isn't it?